weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionPage.tsx

```diff
@@ -327,28 +327,18 @@ const ObjectVersionProducingCallsItem: React.FC<{
   producingCalls: CallSchema[];
   refUri: string;
 }> = props => {
-  if (props.producingCalls.length === 1) {
-    const call = props.producingCalls[0];
-    const {opVersionRef, spanName} = call;
-    if (opVersionRef == null) {
-      return <>{spanName}</>;
-    }
-    return (
-      <CallLink
-        entityName={call.entity}
-        projectName={call.project}
-        opName={spanName}
-        callId={call.callId}
-        variant="secondary"
-      />
-    );
+  const call = props.producingCalls[0];
```

Comment:
Oh nice call. Yeah, i was under the impression that it would be extremely rare to have multiple. However, given your question, that caused me to question this myself. Added back this functionality.

Reply:
Yeah, I originally was thinking of "produced" as "created" and was a little surprised there could be multiple, but it made more sense when I thought of it as "return value".